app/psc/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+edition = "2018"
+readme = "README.md"
+name = "psc"
+version = "0.1.0"
+
+[features]
+itm = ["panic-itm", "kern/klog-itm"]
+semihosting = ["panic-semihosting", "kern/klog-semihosting"]
+
+[dependencies]
+cortex-m = { version = "0.7", features = ["inline-asm"] }
+cortex-m-rt = "0.6.12"
+panic-itm = { version = "0.4.1", optional = true }
+panic-halt = { version = "0.2.0", optional = true }
```

Comment:
There's no actual way to turn this on by feature, you can remove it.

Reply:
Fixed